compiler/testData/codegen/box/delegation/delegationToVal.kt

```diff
@@ -1,4 +1,5 @@
 // IGNORE_BACKEND_FIR: JVM_IR
+// IGNORE_BACKEND: JVM_IR
```

Comment:
This test should not have passed. It is checking that no delegate field is created for interface delegation. It is doing so based on the name of the field, and therefore it now correctly fails because we do not have that optimization.